src/Carter.Analyzers/Carter.Analyzers.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
should this be net8.0?

Reply:
It could be, however I'd advise against it if you want maximum compatibility. Roslyn components are supposed to target netstandard2.0. I've noticed targeting higher versions works too, however not for everybody. Rider always works fine, it might just not work for some Visual Studio users. For most Visual Studio users it will also be fine. 

Considering you want to ship this with the Carter package, I say go for it and if we notice that the analyzer does not kick in for the majority of users, we can still separate this into a separate package. I will make the necessary adjustments.